.tools/test/DEPLOYMENT.md

```diff
@@ -1,92 +1,100 @@
 # Deployment Instructions
 
-There are two ways to deploy the code in this directory.
+This repository contains infrastructure deployment scripts for testing SDK example code. The infrastructure is managed through AWS CDK and can be deployed in two ways:
+1. [deploy.py](#1-using-the-deploy-script)
+2. [invoking the CDK directly](#2-invoking-cdk-directly)
 
-## 1. Using the deploy script
+## Option 1. Using `deploy.py`
 
-To deploy any stack in this directory, run the [deploy.py](stacks/deploy.py) script.
+The [deploy.py](stacks/deploy.py) script is the primary method for deploying the infrastructure stacks.
+
+It is designed to run from MacOS terminal.
+
+It uses Python's `subprocess` module to execute CDK commands (as CDK doesn't support exist as a Python library). It relies on environment variables (noted in this document) throughout the deployment process.
+
+The script handles three types of deployments:
+
+1. **Images Stack** (`images`):
+   - Creates empty ECR private repositories for all tools listed in [targets.yaml](stacks/config/targets.yaml)
+   - Users must implement their own image versioning and pushing mechanism
+   - Example: GitHub Actions with OIDC provider works well for this purpose
+
+2. **Admin Stack** (`admin`):
+   - Deploys event emission infrastructure
+   - Creates IAM policies for cross-account event subscription
+   - **Required**: Must be deployed before any plugin stacks
+   - Works with single or multiple accounts listed in [targets.yaml](stacks/config/targets.yaml)
+
+3. **Plugin Stack** (`plugin`):
+   - Deploys two stacks to each account in [targets.yaml](stacks/config/targets.yaml):
+     1. Plugin stack that subscribes to admin stack events
+     2. Account nuker stack that cleans up residual test resources
+   - Requires `admin` stack to be deployed first
 
 ### Script Prerequisites
 
+- MacOS terminal environment
```

Comment:
It is designed to run from the command line interface (CLI) on macOS or Linux systems. You can use the default terminal emulator on macOS, such as zsh or bash, or any other terminal emulator of your choice AFAIK.

Reply:
I just wouldn't say "MacOS terminal environment". It's not a specific enough prerequisite to be useful. If you add more specifics, it will then become completely unhelpful.